numpy/core/tests/test_indexing.py

```diff
@@ -15,7 +15,14 @@
 
 
 try:
-    cdll = np.ctypeslib.load_library('multiarray', np.core.multiarray.__file__)
+    cdll = None
+    if hasattr(sys, 'gettotalrefcount'):
```

Comment:
A note that this is a check for running in a debug version of Python would be helpful.